src/components/new-members/index.js

```diff
@@ -6,23 +6,42 @@ import { membersContext } from '@store/members/members-context';
 import { searchMemberContext } from '@store/search-members/searchMembers-context';
 import { searchMembers } from '@helper-functions/search-members';
 
+const cardColorArray = [
```

Comment:
push this code into a constants file or folder

Reply:
sure